dev/tests/static/testsuite/Magento/Test/Integrity/_files/extension_dependencies_test/extension_conflicts/inventory.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+return [
+    // the following modules must be disabled when Live Search is used
+    // so core modules must not be dependent on them
+    'Magento\LiveSearch' => [
+        'Magento\InventoryElasticSearch',
```

Comment:
Please fix module naming

```suggestion
        'Magento\InventoryElasticsearch',
```